.coveragerc

```diff
@@ -0,0 +1,2 @@
+[run]
+omit = src/ott/types.py
```

Comment:
This is already specified in [pyproject.toml](https://github.com/ott-jax/ott/blob/main/pyproject.toml#L121).